### ISSUE TITLE
Stack trace don`t contain exception message

### ISSUE THREAD

**Issue by hezymal** [onury/jasmine-console-reporter#8]
Stack trace don`t contain exception message. I am use [TypeScript](https://www.npmjs.com/package/typescript).

![1](https://user-images.githubusercontent.com/4828142/37563118-41129e8c-2abd-11e8-922d-a07a2ea6f1ef.png)
![2](https://user-images.githubusercontent.com/4828142/37563119-4919b598-2abd-11e8-8268-4f556f61d02d.png)

if changed index.js to the next, then everything works:
```js
function specFailureDetails(spec, num) {
    print.line(style.red(num + ') '));
    const title = spec.fullName.replace(spec.description, ': ' + spec.description);
    print.str(style.cyan(title));
    let i, failedExpectation, errInfo;
    for (i = 0; i < spec.failedExpectations.length; i++) {
        failedExpectation = spec.failedExpectations[i];
        errInfo = 'Error: ' + (failedExpectation.message || 'Unknown Error');
        errInfo = style.red(errInfo);
        if (failedExpectation.stack) {
            errInfo += utils.reStack(failedExpectation.stack, options.cleanStack, style);
        } else {
        }
        print.line(utils.indent(errInfo, INDENT_UNIT));
    }
    print.newLine();
}
```

nodejs v9.3.0 and yarn.lock:
```bash
"@types/jasmine@^2.8.6":
  version "2.8.6"
  resolved "https://registry.yarnpkg.com/@types/jasmine/-/jasmine-2.8.6.tgz#14445b6a1613cf4e05dd61c3c3256d0e95c0421e"

jasmine-console-reporter@^2.0.1:
  version "2.0.1"
  resolved "https://registry.yarnpkg.com/jasmine-console-reporter/-/jasmine-console-reporter-2.0.1.tgz#9cd85b4ba9bd81dd45817ebe016f283e8dbf6d7b"
  dependencies:
    ansi-styles "^3.2.0"
    chalk "^2.1.0"

jasmine-core@~3.1.0:
  version "3.1.0"
  resolved "https://registry.yarnpkg.com/jasmine-core/-/jasmine-core-3.1.0.tgz#a4785e135d5df65024dfc9224953df585bd2766c"

jasmine@^3.1.0:
  version "3.1.0"
  resolved "https://registry.yarnpkg.com/jasmine/-/jasmine-3.1.0.tgz#2bd59fd7ec6ec0e8acb64e09f45a68ed2ad1952a"
  dependencies:
    glob "^7.0.6"
    jasmine-core "~3.1.0"

typescript@^2.7.2:
  version "2.7.2"
  resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.7.2.tgz#2d615a1ef4aee4f574425cdff7026edf81919836"
```

**Comment by onury** [comment]
Thanks. I've already fixed this. Expect a new release shortly.

Duplicate of #7

**Comment by hezymal** [comment]
Ohh, I'm sorry. I missed it.

**Comment by onury** [comment]
Fixed in v3.0.0
